src/concourse/pipelines/infrastructure/xpro_partner_dns/pulumi_pipeline.py

```diff
@@ -0,0 +1,37 @@
+from concourse.lib.constants import PULUMI_CODE_PATH
```

Comment:
Not really sure about the filename itself `pulumi_pipeline.py`

Reply:
Yeah, this is in keeping with a loose convention that has been growing organically instead of intentionally. Probably something worth discussing.